index.css

```diff
@@ -222,3 +220,748 @@ section img {
   top: 4px;
   left: 0;
 }
+
+.recent {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 120px 0;
+  gap: 80px;
+  position: absolute;
+  width: 375px;
+  height: 3455px;
+  left: 0;
+  top: 900px;
+}
```

Comment:
- [ ] As the last code reviewer said: It looks like the contents of the section of your work are aligned to the left. kindly align them to the center to match the design template. You are expected to stick to Figma design as much as possible. Check the images below. 👇🏽 
- [ ] **Kindly Fix your button size** to look similar to the Figma template.

- **Your design**

![image](https://user-images.githubusercontent.com/58642949/192075646-dbfd41e0-3c61-46e5-8f38-7ba256172611.png)

- **Figma design**
![image](https://user-images.githubusercontent.com/58642949/192075690-503471ea-1dc1-4795-812e-a049a68ccf8c.png)